contents/dataframesmeta_chain.md

```diff
@@ -0,0 +1,67 @@
+## Piping Operations {#sec:dataframesmeta_chain}
+
+R users are familiar with the pipe operator `%>%`.
+It allows you to **chain operations together**.
+That means that the output of an operation will be used as input in the next operation and so on.
+
+This can be accomplished with the `@chain` macro.
+The way it works is quite simple.
```

Comment:
```suggestion
```
It's up for the reader to decide whether something is simple. Maybe they find it not simple and get frustrated